src/index.html

```diff
@@ -17,16 +17,21 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form data-qa="big">
+      <input
+        class="icon search-bar"
+        type="search"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
+    <form data-qa="small">
+      <input
+        class="icon search-bar"
+        type="search"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
```

Comment:
The input element in the second form is missing the `data-qa` attribute. Please add this attribute to maintain consistency and for potential testing purposes.